### ISSUE TITLE
similar courses and programs

### ISSUE THREAD

**Issue by SaraMati** [UofTCoders/grad-course#4]
- [Department of Biomedical data science, Standford](http://med.stanford.edu/dbds/education/workshop.html)
- [Johns Hopkins Biomedical data science](https://www.bme.jhu.edu/graduate/mse/degree-requirements/biomedical-data-science/)
- [Johns Hopkins, Neuro Data course](https://neurodatadesign.github.io/Syllabus/#)
- [Joshua T. Vogelstein talk](https://neurodatadesign.github.io/Syllabus/)
- [Johns Hopkins Neuroengineering](https://www.bme.jhu.edu/graduate/mse/degree-requirements/neuroengineering-and-technology/)
- [University of Rochester courses for Biomedical Data Science](https://www.urmc.rochester.edu/biomedical-data-science/curriculum.aspx)
- [Biomedical Data Science course, Edinburgh](http://www.drps.ed.ac.uk/17-18/dpt/cxmath11174.htm)
- https://neurohackademy.org/
- [UBC stat course](https://stat545.com/)
- [UBC Statistical Methods for High Dimensional Biology](https://stat540-ubc.github.io/)
- https://github.com/eacton/CAGEF

**Comment by linamnt** [comment]
This is a similar course in IMS, MSC1090 "Computational Biostatistics in R": https://support.scinet.utoronto.ca/education/go.php/399/index.php/ib/1/p_course/399/ib/1/

**Comment by lwjohnst86** [comment]
@linamnt nice resource, though I'm really not a fan of putting code on slides to go over them. Code-alongs are better for learning and retention.

**Comment by linamnt** [comment]
Yes definitely, I think thats a great way to distinguish ourselves from that course.

**Comment by lwjohnst86** [comment]
@linamnt plus we'll have a much much much nicer website :wink: well, we'll have a website, full stop :wink:

**Comment by SaraMati** [comment]
I liked the material on 'best practices' lecture. and the good code/bad code part can be done in code-along format
